### PR TITLE
Languages, Frame work and Contact

### PR DESCRIPTION
**Pull request by solomonetchu**
This request has changes made at level of the Languages, and  Contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -304,21 +304,169 @@ <h5 class="work__Title--text">Multi-Post Stories Gain&plus;Glory</h5>
...
+                            <label for="contactEmail"></label>
+                            <input id="contactEmail" name="Email" required="" type="email" placeholder="Email adress"/>
+                            <label for="TextArea"></label>
+                            <textarea id="TextID" name="TextArea" minlength="15" maxlength="300"  required rows="7" placeholder="Enter Text"></textarea>
+    
+                            <div class="btn__style fotobg">
+                    
+                                <a href="#">
+                                    <button class="Abtn btn">Get in Touch
+                                        <!-- <div class="bgtp"><img src="/img/bgAbout_meDown.png" alt="About me background" class="bgtp__img"></div> -->
```
Comment (1GORDON) [inline index.html]: Great job @solomonetchu 🤝

- [ ] Kindly remove all commented code from your project as this is regarded as a bad practice.

File: index.html [hunk 2]
```diff
@@ -304,21 +304,169 @@ <h5 class="work__Title--text">Multi-Post Stories Gain&plus;Glory</h5>
...
+                    </p>
+                </div>
+    
+    
+                <div class="contact__container">
+    
+                    <div class="contact-us">
+                        <form action="#"name="simple-contact-form" accept-charset="utf-8" method="post">
+                            <label for="FullName"></label>
+                            <input id="FullName" name="Full Name" required="" type="text" placeholder="Full name" />
```
Comment (1GORDON) [inline index.html]: Great job @solomonetchu 🤝

- [ ] As stated in the project requirements, kindly add the `maxlength="30"` property to the name input field.
- [ ] You may also remove the `value` from the `required` property.

File: index.html [hunk 3]
```diff
@@ -304,21 +304,169 @@ <h5 class="work__Title--text">Multi-Post Stories Gain&plus;Glory</h5>
...
+                            <label for="contactEmail"></label>
+                            <input id="contactEmail" name="Email" required="" type="email" placeholder="Email adress"/>
+                            <label for="TextArea"></label>
+                            <textarea id="TextID" name="TextArea" minlength="15" maxlength="300"  required rows="7" placeholder="Enter Text"></textarea>
```
Comment (1GORDON) [inline index.html]: Great job @solomonetchu 🤝

- [ ] Kindy make sure the `maxlenght="500"` property of the text area is as stated in the project requirements.

File: index.html [hunk 4]
```diff
@@ -304,21 +304,169 @@ <h5 class="work__Title--text">Multi-Post Stories Gain&plus;Glory</h5>
...
+                    <p class="FormText__para">
+                        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                    </p>
+                </div>
+    
+    
+                <div class="contact__container">
+    
+                    <div class="contact-us">
+                        <form action="#"name="simple-contact-form" accept-charset="utf-8" method="post">
```
Comment (1GORDON) [inline index.html]: Great job @solomonetchu 🤝

- [ ] Kindly make sure the form has the action property of `action="https://formspree.io/f/{form_id}"` just like it is stated in the project requirements.

File: index.html [hunk 5]
```diff
@@ -304,21 +304,169 @@ <h5 class="work__Title--text">Multi-Post Stories Gain&plus;Glory</h5>
...
+        <footer>
+            <div class="footer__container">
+                <div class="FormText">
+                    <p class="FormText__para">
+                        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                    </p>
+                </div>
+    
+    
+                <div class="contact__container">
```
Comment (1GORDON) [inline index.html:399]: Great job @solomonetchu 🤝

- [ ] Kindly make sure the elements of this container are layout with `display: flex`.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @solomonetchu ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [ ] You may always write your commit messages using imperative tense something like `Add Linter Files` instead of `Adding Linter Files`.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @solomonetchu , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**